slang-gfx.h

```diff
@@ -1470,6 +1477,8 @@ class IDevice: public ISlangUnknown
         SlangDesc slang = {};
     };
 
+    virtual SLANG_NO_THROW Result SLANG_MCALL getNativeHandle(Desc::ExistingDeviceHandles* outHandle) = 0;
```

Comment:
It will be good to move ExistingDeviceHandle to be directly inside IDevice, and just rename it to NativeHandle.